src/lib/emoticons.js

```diff
@@ -32,7 +32,7 @@ export function checkWhitespace(node, rangeHelper) {
 		var range = rangeHelper.cloneSelected();
 		var rangeStart = -1;
 		var rangeStartContainer = range.startContainer;
-		var previousText = prev.nodeValue || '';
+		var previousText = (prev && prev.nodeValue) || '';
```

Comment:
It might be worth doing a breaking change to up the target JS, it's getting annoying not being able to use features like optional chaining plus it would be nice to be able to tidy up the code.